osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -24,6 +29,32 @@ private void load(OsuColour colours)
             AddButton("Continue", colours.Green, () => OnResume?.Invoke());
             AddButton("Retry", colours.YellowDark, () => OnRetry?.Invoke());
             AddButton("Quit", new Color4(170, 27, 39, 255), () => OnQuit?.Invoke());
+
+            AddInternal(pauseLoop = new SkinnableSound(new SampleInfo("pause-loop"))
+            {
+                Looping = true,
+            });
+            // PopIn is called before updating the skin, and when a sample is updated, its "playing" value is reset
+            // the sample must be played again(and if it plays when it shouldn't, the volume will be at 0)
+            pauseLoop.OnSkinChanged += () => pauseLoop.Play();
+        }
+
+        protected override void PopIn()
+        {
+            base.PopIn();
+
+            //SkinnableSound only plays a sound if its aggregate volume is > 0, so the volume must be turned up before playing it
+            pauseLoop?.TransformBindableTo(pauseLoop.Volume, 0.00001);
+            pauseLoop?.TransformBindableTo(pauseLoop.Volume, 1.0f, 400, Easing.InQuint);
+            pauseLoop?.Play();
+        }
+
+        protected override void PopOut()
+        {
+            base.PopOut();
+
+            pauseLoop?.Stop();
+            pauseLoop?.TransformBindableTo(pauseLoop.Volume, 0.0f);
```

Comment:
why is this required? what is it actually doing?

Reply:
This one is called when the user want to continue.
Since we exit the pause screen, we want to stop the loop, and its volume is muted to properly perform the volume fading in the next PopIn call.
I guess that since we have to change the volume before playing the loop, it's unneeded, and changing it to something different than 0 would only confuse peoples, so I will remove it.

Reply:
it should probably be faded out and then stopped.

Reply:
It seems that the fading stops when the pause overlay is hidden, so i can't use the same fading as the one used when the overlay appears.
On osu!Stable, there is no fading out, but it's hidden by a loud sound when pressing continue.
For now an OutQuad easing of 190ms sounds okay.